pkg/integrations/v4/supervisor_fb.go

```diff
@@ -144,6 +187,80 @@ func saveToTempFile(config []byte) (string, error) {
 	return file.Name(), nil
 }
 
+// keeps the most recent config files, up to maxNumberOfFbConfigTempFiles, and removes the rest
+func removeFbConfigTempFiles(maxNumberOfFbConfigTempFiles int) ([]string, error) {
+
+	files, err := os.ReadDir(os.TempDir())
+	if err != nil {
+		return nil, err
+	}
+
+	var fbConfigTempFiles []fs.DirEntry
+
+	for _, file := range files {
+		if !file.IsDir() && strings.HasPrefix(file.Name(), "nr_fb_config") {
+			fbConfigTempFiles = append(fbConfigTempFiles, file)
+		}
+	}
+
+	if len(fbConfigTempFiles) <= maxNumberOfFbConfigTempFiles {
+		return nil, nil
+	}
+
+	// sort fbConfigTempFiles by ascending modification date
+	sort.Slice(fbConfigTempFiles, func(i, j int) bool {
+		fileInfo1, _ := fbConfigTempFiles[i].Info()
+		fileInfo2, _ := fbConfigTempFiles[j].Info()
+		return fileInfo1.ModTime().Before(fileInfo2.ModTime())
+	})
+
+	var removedConfigTempFiles []string
+	var configTempFilesToRemove []string
+	var errors ErrorsList
+
+	// create list of fbConfigTempFiles to remove
+	for i := 0; i < len(fbConfigTempFiles)-maxNumberOfFbConfigTempFiles; i++ {
+		configTempFilesToRemove = append(configTempFilesToRemove, fbConfigTempFiles[i].Name())
+	}
+
+	// extract lua filter filenames from config temp files to remove
+	for _, fbLuaFilterTempFilename := range configTempFilesToRemove {
+		if fbLuaFilterTempFilenames, err := extractLuaFilterFilenames(fbLuaFilterTempFilename); err != nil {
+			errors.Add(err)
+		} else {
+			configTempFilesToRemove = append(configTempFilesToRemove, fbLuaFilterTempFilenames...)
+		}
+	}
+
+	// remove all config and lua filter temp files from temporary directory
+	for _, fbLuaFilterTempFilename := range configTempFilesToRemove {
```

Comment:
I would rename `fbLuaFilterTempFilename`  to `configTempFileToRemove`☝️

Reply:
Sure!